File: YoutubeClone/DTO/VideoSummary.cs
namespace YoutubeClone.DTO
{
    public class VideoSummary
    {
        public Guid Id { get; set; }
        public Guid ChannelId { get; set; }
        public string ChannelName { get; set; } = string.Empty;
        public string ChannelThumbnailUrl { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string ThumbnailUrl { get; set; } = string.Empty;
        public string Url { get; set; } = string.Empty;
        public long Views { get; set; }
        public long Likes { get; set; }
        public long Dislikes { get; set; }
        public DateTime DateCreated { get; set; }
    }
}

File: YoutubeClone/ClientApp/src/video-summary.ts
export default interface VideoSummary {
    id: string;
    channelId: string;
    channelName: string;
    channelThumbnailUrl: string;
    name: string;
    description: string;
    thumbnailUrl: string;
    url: string;
    views: number;
    likes: number;
    dislikes: number;
    dateCreated: string;
}